desktop/libs/aws/src/aws/s3/upload.py

```diff
@@ -43,10 +44,79 @@
 else:
   from django.utils.translation import ugettext as _
 
-DEFAULT_WRITE_SIZE = 1024 * 1024 * 50  # TODO: set in configuration (currently 50 MiB)
+DEFAULT_WRITE_SIZE = 1024 * 1024 * 128  # TODO: set in configuration (currently 50 MiB)
```

Comment:
Change the value on the comment also.

Also maybe set this as a config like said in the comment too?